src/ood-gen/bin/gen.ml

```diff
@@ -10,8 +10,8 @@ let term_templates =
     ("success_story", Ood_gen.Success_story.template);
     ("tool", Ood_gen.Tool.template);
     ("tutorial", Ood_gen.Tutorial.template);
-    ("workshops", Ood_gen.Workshop.template);
```

Comment:
I think something has gone when your were managing the conflicts during the rebase, did you just accept your changes perhaps? It has removed the workshops from here.